rpc/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rpc"
```

Comment:
It's because this `rpc` crate is only related to Trin, and I don't want to duplicate all trin folders to include `trin-...` because I think this is redundant.